### PR TITLE
Global disable web page preview parameter

### LINKED ISSUE #690: Global disable_web_page_preview parameter
**Is your feature request related to a problem? Please describe.**
Set `disable_web_page_preview` in `aiogram.Bot`:
`bot = aiogram.Bot(token=..., disable_web_page_preview=...)`

**Describe the solution you'd like**
Like logic with `parse_mode`, but with `disable_web_page_preview` (https://github.com/aiogram/aiogram/blob/358ecc78213183adeb70809d7f3e95f1524657fb/aiogram/bot/bot.py#L311-L312):
```
if self.disable_web_page_preview and entities is None:
    payload.setdefault('disable_web_page_preview', self.disable_web_page_preview)
```
   
In ideal, need paste that piece of code to all methods with messages where supports `disable_web_page_preview`

### PR DESCRIPTION
**Pull request by samuelfirst**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes #690

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: aiogram/bot/bot.py [hunk 1]
```diff
@@ -310,6 +310,8 @@ async def send_message(self,
         payload = generate_payload(**locals())
         if self.parse_mode and entities is None:
             payload.setdefault('parse_mode', self.parse_mode)
+        if self.disable_web_page_preview and entities is None:
```
Comment (evgfilim1) [inline aiogram/bot/bot.py:313]: I don't understand why you check whether entities are not specified. It's needed only for `parse_mode`.

```suggestion
        if self.disable_web_page_preview:
```

File: aiogram/bot/bot.py [hunk 2]
```diff
@@ -2408,6 +2409,8 @@ async def edit_message_text(self,
...
```
Comment (evgfilim1) [inline aiogram/bot/bot.py:2412]: Same here

```suggestion
        if self.disable_web_page_preview:
```

**Review by evgfilim1 — CHANGES_REQUESTED** [review]

File: aiogram/bot/bot.py (hunk 1 above)
Comment (samuelfirst) [reply aiogram/bot/bot.py]: done! fixed

File: aiogram/bot/bot.py (hunk 2 above)
Comment (samuelfirst) [reply aiogram/bot/bot.py]: fixed

**Review by evgfilim1 — APPROVED** [review]
Looks good to me, thanks for PR!

**Comment by evgfilim1** [comment]
@samuelfirst have you tested this change? I don't see it in your pull request description.

**Review by Olegt0rr — APPROVED** [review]

**Comment by samuelfirst** [comment]
@evgfilim1 yep, I tested. It works well

**Pull request merged**